src/styles/utils/extend.scss

```diff
@@ -0,0 +1,23 @@
+%clock-center {
+  height: $clock-base-size / 30;
+  width: $clock-base-size / 30;
+
+  border-radius: $clock-base-size / 60;
+  z-index: 1;
+}
+
+%second-hand-position {
+  height: $clock-base-size / 2.5;
+  width: $clock-base-size / 100;
+
+  bottom: $clock-base-size / 2;
+  border-radius: $clock-base-size / 120;
+}
+
+%minute-hand-position {
+  height: $clock-base-size / 4;
+  width: $clock-base-size / 60;
+
+  bottom: $clock-base-size / 2;
+  border-radius: $clock-base-size / 120;
+}
```

Comment:
You can make mixin instead of these two

Reply:
I didn't use extends before. So I wanted to use it)